src/rf/js/src/core/models.js

```diff
@@ -60,7 +60,7 @@ var Layer = Backbone.Model.extend({
         return _.findWhere(this.get('images'), {id: id});
     },
 
-    isUploading: function() {
+    isPreValidated: function() {
```

Comment:
preValidated means it's before (image) validation has occurred. If I used isStarted() or isCreated() it's also confusing because one would expect that for layers with status==processing, isStarted()==true (after all, they have started), but that would not be the case.